target_object_database/forms.py

```diff
@@ -95,9 +95,9 @@ def handle_file_upload(data, files, lti_params):
     list_of_files = files.getlist('target_file')
 
     try:
-        image_backend_class = getattr(image_store.backends, settings.IMAGE_STORE_BACKEND)
-        image_backend = image_backend_class(settings.IMAGE_STORE_BACKEND_CONFIG, lti_params)
-        manifest_url = image_backend.store([file for file in list_of_files], title)
+        cls = image_store.backends.get_backend_class(settings.IMAGE_STORE_BACKEND)
+        image_backend = cls(config=settings.IMAGE_STORE_BACKEND_CONFIG, lti_params=lti_params)
+        manifest_url = image_backend.store(list_of_files, title)
```

Comment:
It's interesting that this was changed back to having just `list_of_files` passed, rather than having to unpack it as a list comprehension. That was something that I did when resolving a bug around uploading multiple files. I just tested on dev, and multiple image uploads seem to be working just fine. Is this now being handled in the sdk?

I remember the issue stemmed `list_of_files` being a `MultiValueDict` rather than a list or something like that.

Reply:
@dodget Great question. The list comprehension is now happening in the `store()` method of the image store backend, which is a result of some changes to the sdk. Here's where it's happening now:

https://github.com/Harvard-ATG/annotationsx/blob/63adf774428f7b1883a97506150dbed59f5fcc18/image_store/backends.py#L93-L97

Although, I think you are I should restore the list comprehension here so that there's no surprise down the line (or even if this code should change and we don't realize that it is in fact a `MultiValuedDict` - I'm sure I'll forget). 

I'm going to restore the list comprehension here with a comment to that effect. Thanks for spotting that.